scalding-parquet-scrooge-fixtures/src/test/resources/test.thrift

```diff
@@ -86,14 +86,6 @@ struct StructWithReorderedOptionalFields {
   1: optional i32 fieldOne,
 }
 
-struct TestPersonWithRequiredPhone {
```

Comment:
The scrooge generator was cribbing cause this was being declared twice. The old version seem to not mind.